packages/libs/src/sdk/services/Solana/programs/BaseSolanaProgramClient.ts

```diff
@@ -90,14 +105,41 @@ export class BaseSolanaProgramClient {
       instructions,
       feePayer,
       recentBlockhash,
-      addressLookupTables = []
+      addressLookupTables = [],
+      priorityFee
     } = await parseParams('buildTransaction', BuildTransactionSchema)(params)
 
     if (!recentBlockhash) {
       const res = await this.connection.getLatestBlockhash()
       recentBlockhash = res.blockhash
     }
 
+    if (priorityFee !== undefined) {
+      if ('microLamports' in priorityFee) {
```

Comment:
Wonder if we should also check the type/value of `microLamports` and give the user a helpful exception?
Or is TS checking enough?

Reply:
zod has us covered!